xbmc/interfaces/legacy/ListItem.h

```diff
@@ -291,6 +291,40 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
       ///
       /// \ingroup python_xbmcgui_listitem
+      /// @brief \python_func{ setRating(key, value) }
+      ///-----------------------------------------------------------------------
+      /// Sets a listitem's rating. It needs at least a rating in the dict
+      ///
+      /// @param key    string - the type of the rating. Any string.
+      /// @param key    dictionary - `{ default:bool, rating:float, votes:int}`.
+      ///  - Some example type (any string possible):
+      ///  | Label         | Type                                              |
+      ///  |:-------------:|:--------------------------------------------------|
+      ///  | imdb          | string - rating type
+      ///  | tvdb          | string - rating type
+      ///  | tmdb          | string - rating type
+      ///  | anidb         | string - rating type
+      ///
+      ///
+      ///
+      ///-----------------------------------------------------------------------
+      ///
+      /// **Example:**
+      /// ~~~~~~~~~~~~~{.py}
+      /// ...
+      /// # setRating(values)
+      /// self.list.getSelectedItem().setRating({ 'default':True, 'rating':9.5, 'votes':4}, 'imdb')
```

Comment:
Documentation and implementation says different things. If there's only 3 possible keys though, im not sure it makes sense to have a dictionary for that. Just have them as regular parameters.